### PR TITLE
Bug/mc 17  no results message

### PR DESCRIPTION
**Pull request by MatejaC**

### CODE REVIEW

File: src/components/SearchBox.tsx [hunk 1]
```diff
@@ -53,14 +53,12 @@ const SearchBox = ({ setSearchResults }: SearchBoxProps) => {
         setHelperText('');
       }
     } catch (error) {
-      console.error('Error fetching data:', error);
-      alert('Something went wrong. Please try again.');
+      setHelperText('Something went wrong. Please try again.');
       setHelperText('No results match criteria.');
```
[the remark this answers is not in the document]
Comment (MatejaC) [reply src/components/SearchBox.tsx:57]: if alert is too much,  what's the solution?

[the remark this answers is not in the document]
Comment (MatejaC) [reply src/components/SearchBox.tsx:57]: oh, I see

[the remark this answers is not in the document]
Comment (e1401) [reply src/components/SearchBox.tsx:57]: `  setHelperText('No results match criteria.');` can be removed, yes

**Review by e1401 — APPROVED** [review]
🚀

**Pull request merged**